packages/contracts/deploy/helpers.ts

```diff
@@ -117,6 +120,20 @@ export async function createPluginRepo(
   releaseMetadata: string,
   buildMetadata: string
 ): Promise<void> {
+  const {network} = hre;
+  const pluginDomain =
+    process.env[`${network.name.toUpperCase()}_PLUGIN_ENS_DOMAIN`] || '';
+  if (
+    await isENSDomainRegistered(
+      `${pluginContractName}.${pluginDomain}`,
+      ENS_ADDRESSES[network.name]
```

Comment:
the domain passing is correct here, but `ENS_ADDRESSES[network.name]` not.

Reply:
fixed